x12-edi/2.5/modules/ROOT/pages/index.adoc

```diff
@@ -26,124 +26,127 @@ To use the X12 EDI connector in a production environment, you must have:
 |===
 
 TA1 interchange acknowledgments are in the form of maps representing the TA1 segment data and linked to data on the corresponding interchange:
 
 [%header%autowidth.spread]
 |===
 |Key name |Description
-|Interchange |Map of ISA interchange header segment values. For TA1 acknowledgments generated by receive processing, this map is a copy of the data for the interchange matching the TA1 with sender and receiver identification components (ISA05/ISA06 and ISA07/ISA08) interchanged. When writing a message, these values are used for creating the enclosing interchange (gathering transactions with the same interchange values into a single interchange, regardless of whether the actual maps are the same or not). Values not populated in this map default to the values from the Interchange map at the message level.
+|Interchange |Map of ISA interchange header segment values. For TA1 acknowledgments generated by receive processing, this map is a copy of the data for the interchange matching the TA1 with sender and receiver identification components (ISA05/ISA06 and ISA07/ISA08) interchanged. When writing a message, these values are used for creating the enclosing interchange. That is, gathering transactions with the same interchange values into a single interchange, regardless of whether the actual maps are the same. Values not populated in this map default to the values from the Interchange map at the message level.
 |===
 
-== About Time Fields
-
-All X12 Time fields are represented as Integer values for the number of milliseconds.
-For example, if a message has the value `024020`, its DataWeave mapping
-appears as `9620000` milliseconds.
-
-The value `024020` is in the format "HHMMSS", so to convert it to milliseconds, use this formula:
-
-----
-(02*3600000) + (40*60000) + (20*1000) = 9620000
-----
 
 == Exchange Templates and Examples
 
 https://www.mulesoft.com/exchange/[Anypoint Exchange] provides templates
 that you can use as starting points for your apps and examples that illustrate a complete solution.
 
-////
-List and link to up to 10 Exchange templates and examples.
-Use the Integration Pattern categories (broadcast, migration, bidirectional sync, aggregation).
-////
+=== X12 Connector Templates in Exchange
+
+https://www.mulesoft.com/exchange/org.mule.templates/template-b2b-edi-outbound-x12-map/[Map application messages to outbound EDI X12 transactions]
+
+Use this for setting up your Outbound transactions.
```

Comment:
Can you say "Sets up your Outbound transactions" instead?

Reply:
Fixed.